template/scripts/drupal-vm/drupal-vm.drushrc.aliases.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+// [vagrant_machine_name].local
+$aliases['${project.machine_name}.local'] = array(
```

Comment:
(Plus it follows the convention for the other environments, of [machine].local, .dev, .prod, etc.)